concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1554,7 +1554,7 @@ jobs:
                 . ./config/config.sh
                 echo | cf login -a "${API_ENDPOINT}" -u "${CF_ADMIN}" -p "${CF_PASS}"
 
-                if cf service-brokers | grep "rds-broker\s"; then
+                if cf service-brokers | grep "rds-broker\\s"; then
```

Comment:
Assuming shellcheck is happy with it, it feels cleaner (and more obvious to the next dev) to single-quote these, so we can drop the double-backslash. Ditto every other `grep "string with no variables"`, below.